android/daemon/src/main/java/org/mozilla/firefox/vpn/daemon/VPNService.kt

```diff
@@ -588,6 +589,16 @@ class VPNService : android.net.VpnService() {
         return confBuilder.build()
     }
 
+    private fun getServerConfig(obj: JSONObject, useFallbackServer: Boolean): JSONObject {
```

Comment:
Just a minor nitpick, do we really need the `useFallbackServer` parameter anymore? That seems like it should be covered by the `currentServerConfig` variable.

Also, this function is not idempotent, it actually does the rotation between servers, so a better name might be `getNextServerConfig()` to imply that it's iterating.

Reply:
Updated to `getNextServerConfig`

`useFallbackServer` now just controls whether we use the first server in the config (when false) or the next server in line (when true). I think using the client-selected main server on initial connection seems helpful - and we thus would need to keep `useFallBackServer` in order to reset `currentServerConfig`. (If the daemon is maintained between VPN activations - I'm not sure if it is - we'd otherwise never reset `currentServerConfig` back to 0, and potentially take a non-first server on initial connection to a new geo, for instance.)